install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -454,6 +454,10 @@ data:
   #  - portmap (Enables HostPort support for Cilium)
   cni-chaining-mode: {{ .Values.cni.chainingMode }}
 
+  # In cni chaining mode, the other chained plugin is responsible for underlying connectivity,
+  # so cilium eBPF host routing shoud not work, and let it fall back to the legacy routing mode
+  enable-host-legacy-routing: "true"
```

Comment:
We should not have the same key in the configmap twice, I recommend adding this as an `{{ else if ne .Values.cni.chainingMode "none" }}` line here instead.

https://github.com/cilium/cilium/blob/2cc23842f91aa1f593013c5a08afe3b4650cef85/install/kubernetes/cilium/templates/cilium-configmap.yaml#L288-L290

Reply:
ok, i will fix it